app/validation/validator.py

```diff
@@ -277,6 +348,17 @@ def _get_answers_for_block(block_json):
                 answers.append(answer_json)
         return answers
 
+    @staticmethod
+    def _get_referenced_answer_id(answer):
+        referenced_answer_id = None
+
+        if answer.get('max_value') and 'answer_id' in answer.get('max_value'):
+            referenced_answer_id = answer['max_value']['answer_id']
+        if answer.get('min_value') and 'answer_id' in answer.get('min_value'):
+            referenced_answer_id = answer['min_value']['answer_id']
```

Comment:
What should happen if both min and max values reference another answer? If we're not going to support this we should add validation to stop it.

Reply:
@ajmaddaford worth doing a test case to see, are we also referencing from two different answers, or the same answer?